packages/ag-charts-enterprise/src/features/zoom/zoomRange.ts

```diff
@@ -1,9 +1,13 @@
-import { _ModuleSupport } from 'ag-charts-community';
+import { _ModuleSupport, _Util } from 'ag-charts-community';
 
 const { AND, DATE, NUMBER, OR, ObserveChanges, Validate } = _ModuleSupport;
+const { Logger } = _Util;
 
 export class ZoomRange {
     @ObserveChanges<ZoomRange>((target, start) => {
+        if (target.initialStart == null) {
+            Logger.warnOnce('Property [zoom.rangeX] is deprecated. Use [initialState.zoom.rangeX] instead.');
+        }
```

Comment:
This check stops it from warning when the range is restored from `initialState.zoom` as `ZoomRange.initialStart` is set in `ZoomRange.reset()` which is called by `Zoom.onRestoreZoom()`.